.stylelintignore

```diff
@@ -1,4 +1,5 @@
 **/*.min.css
+**/*.test.scss
```

Comment:
Maybe a better way could be to disable stylelint inside the file only. I don't have any strong opinion on this subject but felt fine to ignore tests .

Reply:
I just saw that in [_utilities.test.scss](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/ouds/main/scss/tests/mixins/_utilities.test.scss#L267) we already have generated content with leading 0, and the linter has been disabled only for the concerned line. Maybe we should stick to this in order to be consistent ? I'll do that until we decide what we choose to do.

Reply:
We talked about this with @julien-deramond and decided to just disable the rule for the corresponding lines with a comment, like it's done in utilities test file.